src/platform/datapath_raw.h

```diff
@@ -155,6 +155,7 @@ typedef struct CXPLAT_SOCKET {
     void* CallbackContext;
     QUIC_ADDR LocalAddress;
     QUIC_ADDR RemoteAddress;
+    BOOLEAN Wildcard;   // Using a wildcard local address
```

Comment:
Shouldn't we be able to get this information from the LocalAddress field? The normal way of using sockaddrs is for (e.g. for IPv4) 0.0.0.0 to mean "wildcard".

Reply:
Yes. This is purely an optimization so you don't have to constantly read the whole address everytime.

Reply:
I'd avoid this type of microoptimization at this stage since we're going to probably change the code multiple times and have to carry this extra logic through each time; but if you insist on this, please at least add a code comment to this field that notes it's an optimization- otherwise it's confusing to see.